src/main/java/mate/academy/spring/Application.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring;
+
+import mate.academy.spring.config.AppConfig;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class Application {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext applicationContext =
+                new AnnotationConfigApplicationContext(AppConfig.class);
+        UserService userService = applicationContext.getBean(UserService.class);
```

Comment:
Do we need this code in the main method? If you want to test the solution - let's write the unit tests